jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -475,23 +475,12 @@ public JaegerSpan start() {
       return jaegerSpan;
     }
 
-    @Override
-    public Scope startActive(boolean finishSpanOnClose) {
```

Comment:
Do we really want to do this? I think the idea of 0.33 was to give users a cleaner API that will facilitate transition to OpenTelemetry, but implementations were still advised to support 0.32 for better backwards compatibility.

cc @carlosalberto

Reply:
Well, we won't even support any of the deprecated members (we throw an exception from these methods in the OT shim), so it's not a big difference.

Buuut if you think API or deployment wise we should nevertheless reference 0.32, then lets talk about it and consider reverting the latest update for this in the OT shim.

Reply:
I think I did not explain myself. Imagine an application X instrumented with 0.32 or even 0.31 API. If we do this PR to jaeger client, then  that application X will no longer be able to upgrade to newer versions of Jaeger client, even though technically there are no major breaking changes. At the same time, another application Y instrumented with 0.33 API would still be able to use Jaeger client compiled against 0.32.

Thus we can maintain broader compatibility of Jaeger client.

Reply:
What are your proposals? Keep the client on 0.32.0 as it is a superset of 0.33.0?

Reply:
I have added the removed methods back. To make sure it works I will add a test submodule for 0.32.0 to assure it works.